### PR TITLE
Add ability to disable bunsen form

### PR DESCRIPTION
**Pull request by job13er**
This is a new #feature#

# CHANGELOG
**Added** `formDisabled` property to pass through to `disabled` on the bunsen form, allowing disabling of the entire form at once.

### CODE REVIEW

File: app/templates/components/frost-modal-input.hbs [hunk 1]
```diff
@@ -36,14 +36,15 @@
...
         inline=true
         onChange=onChange
         onValidation=onValidation
         renderers=renderers
         showAllErrors=showAllErrors
         value=formValue
         validators=validators
-        }}
+      }}
       {{/if}}
```
Comment (job13er) [inline app/templates/components/frost-modal-input.hbs]: Darn, just realized this is not indented properly, guess the linting doesn't catch that :(

**Comment by sandersky** [comment]
:+1:

**Comment by sglanzer** [comment]
@rox163 I think we made a mistake coupling the `bunsen` and modal interface.  The form should be a pure yield, which probably makes this a deprecated project in favor of a standard header/content/footer modal.

If we don't do this then any change to `bunsen` has to be propagated into the model interface.

**Comment by rox163** [comment]
@sglanzer should I look again at making the generic modal then? Well, it wont be totally generic as such because it would still have to have all the styles and transitions to support a form in the content block.

**Comment by sglanzer** [comment]
Right - good catch, if we need the styles (positioning) and transitions specific to forms then we can keep this project, but it should be a basic shell around the generic modal that only provides the styles and the liquid fire map.

**Comment by job13er** [comment]
I was a little surprised that the bunsen form wasn't a yield.

**Pull request merged**